src/components/tracked-controls.js

```diff
@@ -35,6 +36,27 @@ module.exports.Component = registerComponent('tracked-controls', {
 
     this.previousControllerPosition = new THREE.Vector3();
     this.updateGamepad();
+    this.onComponentInitialized = bind(this.onComponentInitialized, this);
+  },
+
+  play: function () {
+    this.el.addEventListener('componentinitialized', this.onComponentInitialized);
+  },
+
+  pause: function () {
+    this.el.removeEventListener('componentinitialized', this.onComponentInitialized);
+  },
+
+  onComponentInitialized: function (evt) {
+    if (evt.detail.name !== 'tracked-controls') { return; }
+    var self = this;
+    Object.keys(this.el.components).forEach(function (name) {
+      const component = self.el.components[name];
+      if (component.controllerPresent) {
+        // Emit controllerpresent event.
+        self.el.emit('controllerpresent', {name: name, component: component});
```

Comment:
> You know the entity because you pass it when you suscbribe to it.

if you have something like that line of code in `oculus-touch-controls` and `vive-controls` :
`sceneEl.systems['tracked-controls'].subscribeToController(el, data, callback);`
what are you thinking goes in data?

I am thinking it may be better to provide id and gamepad/controller reference in `controllerconnected` stage events and let the listener decide whether to inject whatever component(s) like `tracked-controls` rather than trying to come up with a data schema with matching filter code